ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```diff
@@ -153,6 +153,20 @@ tests:
         pod should start\| pod sysctls\| build volumes should mount given secrets
         and configmaps into the build pod
     workflow: openshift-e2e-ibmcloud-heterogeneous
+- as: ocp-e2e-powervs-ovn-multi-ppc64le
+  cron: 20 11,23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      IBMCLOUD_CIS_CRN: 'crn:v1:bluemix:public:internet-svcs:global:a/3c24cb272ca44aa1ac9f6e9490ac5ecd:9b372a78-cec3-45b7-875e-04ba0270c87d::'
```

Comment:
We should use it based on the BASE_DOMAIN not the CRN. It allows a seamless change consistent with other jobs. Let's create a tech debt task to fix this.

Reply:
For now I am sticking to this as the internal UPI automation expects us to set the CRN var, Yes the ideal way would be to deduce it based on domain however I had checked and it seemed more of a additional work (Can improve on this later once we have rest of the flow working)